metadata/UiPathDocumentOCR_CPU__11__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "changeLog":  "",
+    "cpu":  0,
+    "description":  "Machine Learning model(available in Preview) for extracting text and checkboxes/radio buttons from Documents. Please see more details including supported languages and link to Activities guide in the Document Understanding Guide here: https://docs.uipath.com/document-understanding/v2021.4/docs/ml-packages#uipath-document-ocr. For alternative deployment methods in high-volume scenarios see also this page: https://docs.uipath.com/document-understanding/v2021.4/docs/ocr-services",
```

Comment:
nit: missing ` ` (space) before `(available` in `Machine Learning model(available in Preview)`

Reply:
Done!